### ISSUE TITLE
Hyperreflexia of biceps

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5914]
**Preferred term label:** Hyperreflexia of biceps

**Synonyms**

**Definition (free text, please give PubMed ID)** Hyperactive stretch reflex of the biceps.

**Parent term (use hpo.jax.org/app)** Hyperreflexia in upper limbs HP:0007350

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033205
